guardrails/classes/history/call.py

```diff
@@ -361,6 +390,28 @@ def _has_unresolved_failures(self) -> bool:
         # No ReAsks and no unresolved failed validations
         return False
 
+    def _has_resolved_failures(self) -> bool:
+        # Check for unresolved ReAsks
+        if len(self.reasks) > 0:
+            return True
```

Comment:
Should this check go away? I think `Call.reasks` is a computed value that will only have reasks when they haven't been `fixed`